### PR TITLE
Fix styling and functionallity for sorting table

### PR DESCRIPTION
**Pull request by rebekkbb**
Fix styling and functionality for sorting table.

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx [hunk 1]
```diff
@@ -30,12 +30,27 @@ const PatientView = (props: {
...
+                        ' ' +
+                        a.submitted.split(' ')[1],
+                ).unix() -
+                dayjs(
+                    b.submitted.split(' ')[0].split('.')[1] +
+                        b.submitted.split(' ')[0].split('.')[0] +
+                        b.submitted.split(' ')[0].split('.')[2] +
+                        ' ' +
+                        b.submitted.split(' ')[1],
+                ).unix(),
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:53]: I think that maybe for now this is fine, so we don't use much time on this. But I agree this is ugly code haha

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss [hunk 2]
```diff
@@ -83,6 +84,14 @@
         cursor: pointer;
     }
 
+    .ant-table-column-sorter {
+        color: whitesmoke;
+        size: 0.01rem;
```
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss:89]: 1 px = 0.0625rem. Revise

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss [hunk 3]
```diff
@@ -83,6 +84,14 @@
...
+    }
+    .ant-table-column-sort {
+        background-color: inherit;
```
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss:92]: ??? What color is this?

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx (hunk 1 above)
Comment (EnvyPhD) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.tsx:53]: Perhaps atleast move this to a helperfunction or something?

File: src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientQuestionnaire.tsx [hunk 4]
```diff
@@ -25,16 +25,17 @@ const PatientQuestionnaire = ({ patientData, questionnaireResponses, questionnai
...
                 ...dataSource,
                 {
                     id: item.resource.id,
-                    schemaName: qdict.get(
-                        item.resource.questionnaire?.reference
-                            ?.substr(item.resource.questionnaire?.reference?.indexOf('Questionnaire/'))
-                            .split('/')[1],
-                    ),
-                    submitted: dayjs(item.resource.meta?.lastUpdated).format('DD.MM.YYYY - HH:mm').toString(),
+                    schemaName: name ? name : 'Udefinert',
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientQuestionnaire.tsx:37]: Should this be changed to 'Tittel mangler'?

**Pull request merged**